### ISSUE TITLE
Merging 2 10x Multiome (RNA/ATAC) objects

### ISSUE THREAD

**Issue by wht10** [timoast/signac#824]
Hey @timoast, 

Thank you for maintaining the signac tool. 

I am interested in merging 2 10x multiome objects. Combining just RNA assays from 10x runs seems pretty straightforward using the seurat [merge vignette](https://satijalab.org/seurat/articles/merge_vignette.html). 

The [merge vignette for 10x ATAC assays](https://satijalab.org/signac/articles/merging.html), details how to create a common peak set using .bed files. 

I didn't see any .bed files from the cell ranger output, so I am wondering what would be the best way to obtain these.

Is it simply the "Peaks" assay that is extracted from the filtered_feature_bc_matric.h5 cellranger output file, or is there a way it can be extracted from the chromatin assay within the seurat object?

**Comment by timoast** [comment]
To merge the datasets, you just need to define a single set of peaks that you want to use across all of the datasets you're analyzing. Some options for doing that:
* use the peaks from one of the datasets and quantify those in each of the other datasets
* Reduce the peak coordinates from all of the individual datasets (as shown in the merge vignette)
* Call peaks using macs2 with the fragment file from each object as input for peak calling (peak calling vignette [here](https://satijalab.org/signac/articles/peak_calling.html))

**Comment by SidG13** [comment]
Hi @timoast, thanks for the input. I have a similar question, but CellRanger does now provide atac peaks as a bed file (perhaps it's a newer version). Accordingly, I've followed the [ATAC merge vignette](https://satijalab.org/signac/articles/merging.html) to produce a reduced/common peak set which is now a Granges object, per your 2nd point. 

My question is, how do we integrate this dataframe into the combined Seurat object? I've made a combined RNA Seurat object following the [RNA merge vignette](https://satijalab.org/seurat/articles/merge_vignette.html). I'd imagine the next step is to create the chromatin assay for the combined object, but I'm a little unclear as to what form the counts and fragments (when combined across samples) objects takes. Sorry if this has been asked before. I think a multiome merge vignette in the future would be extremely helpful.

Thanks for your support for these tools!
